plugins/temporal/temporal.go

```diff
@@ -46,9 +59,11 @@ func (p *Provider) Serve() chan error {
 	p.serviceClient, err = client.NewClient(client.Options{
 		HostPort:      p.config.Address,
 		Namespace:     p.config.Namespace,
-		DataConverter: NewRRDataConverter(),
+		DataConverter: roadrunner_temporal.NewRRDataConverter(),
```

Comment:
We need to improve it to pack into Payloads directly.

Reply:
Agree, will do in a separate ticket